frontend/src/metabase/visualizations/lib/mapping.js

```diff
@@ -3,55 +3,13 @@ import d3 from "d3";
 
 export function computeMinimalBounds(features) {
   const points = getAllFeaturesPoints(features);
-  const gap = computeLargestGap(points, d => d[0]);
   const [west, east] = d3.extent(points, d => d[0]);
   const [north, south] = d3.extent(points, d => d[1]);
 
-  const normalGapSize = gap[1] - gap[0];
-  const antemeridianGapSize = 180 + west + (180 - east);
-
-  if (antemeridianGapSize > normalGapSize) {
-    return L.latLngBounds(
-      L.latLng(south, west), // SW
-      L.latLng(north, east), // NE
-    );
-  } else {
-    return L.latLngBounds(
-      L.latLng(south, -360 + gap[1]), // SW
```

Comment:
this `-360` is the giveaway. longitude only goes to +/- 180, so no matter what the biggest gap is, this is going to give us a left boundary wider than the globe